docs/source/trainer/file_uploading.rst

```diff
@@ -161,6 +161,30 @@ Weights & Biases Artifacts
     # Train!
     trainer.fit()
 
+Neptune File upload
+^^^^^^^^^^^^^^^^^^^
+
+.. seealso::
+
+    The :class:`~composer.loggers.neptune_logger.NeptuneLogger` API Reference.
+
+.. testcode::
+    :skipif: not _NEPTUNE_INSTALLED
+
+    from composer.loggers import NeptuneLogger
+    from composer import Trainer
+
+    # Configure the Neptune logger
+    logger = NeptuneLogger(
+        upload_checkpoints=True,  # enable logging of checkpoint files
+    )
+
+    # Define the trainer
+    trainer = Trainer(..., loggers=logger)
+
+    # Train
+    trainer.fit()
```

Comment:
```suggestion
```python
    from composer.loggers import NeptuneLogger
    from composer import Trainer

    # Configure the Neptune logger
    logger = NeptuneLogger(
        upload_checkpoints=True,  # enable logging of checkpoint files
    )

    # Define the trainer
    trainer = Trainer(..., loggers=logger)

    # Train
    trainer.fit()
```    
```